pkg/windows/dockerfile

```diff
@@ -83,8 +83,10 @@ CMD powershell.exe -Command Push-Location C:/temp/hubble; \
   Move-Item hubble.conf -Destination ./hubble/dist/hubble/etc/hubble/; \
   Move-Item 'hubble/PortableGit' -Destination './hubble/dist/hubble/' -Force; \
   Move-Item 'C:/ProgramData/chocolatey/lib/NSSM/tools/nssm.exe' -Destination './hubble/dist/hubble/' -Force; \
-  Move-Item 'C:/ProgramData/osquery/osqueryi.exe' -Destination './hubble/dist/hubble/' -Force; \
-  Move-Item 'C:/ProgramData/osquery/osqueryd/osqueryd.exe' -Destination './hubble/dist/hubble/osqueryd/' -Force; \
+  If (Test-Path C:/ProgramData/osquery/osqueryi.exe) {Copy-Item 'C:/ProgramData/osquery/osqueryi.exe' -Destination './hubble/dist/hubble/' -Force} \
```

Comment:
In osquery 3.4.0 , osquery files are being created in Programs Files/osquery/ instead of ProgramData/osquery/.
So, I am checking in ProgramData( first preference) as well as ProgramFiles to see if osquery exists there.
This way, the Dockerfile should work even for older osquery installations.